examples/morpheusvm/storage/storage.go

```diff
@@ -115,17 +115,11 @@ func AddBalance(
 	mu state.Mutable,
 	addr codec.Address,
 	amount uint64,
-	create bool,
 ) (uint64, error) {
-	key, bal, exists, err := getBalance(ctx, mu, addr)
+	key, bal, _, err := getBalance(ctx, mu, addr)
```

Comment:
Do we still need the extra parameter returned by `getBalance` ?

Reply:
https://github.com/ava-labs/hypersdk/blob/a47e198193189142e431bc4275738ae65acdbfe2/examples/morpheusvm/storage/storage.go#L143-L145 It's still used by `SubBalance()`